templates/event.php

```diff
@@ -34,7 +34,9 @@
 	</div>
 	<div class="event-details-left">
 		<div class="event-page-content">
-			<?php echo esc_html( $event_description ); ?>
+			<?php
+				echo wp_kses_post( nl2br( make_clickable( $event_description ) ) );
```

Comment:
Does this also work?
```suggestion
				echo wp_kses_post( wpautop( make_clickable( $event_description ) ) );
```

Reply:
I tested and I think we should use this.